spec/request/csrf_request_spec.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'CSRF scenarios', type: :request do
+  # ActionController::Base.allow_forgery_protection = false in the 'test' environment
+  # We explicity enable it for this spec
+  before(:all) do
+    Settings.sentry.dsn = 'truthy'
+    @original_val = ActionController::Base.allow_forgery_protection
+    ActionController::Base.allow_forgery_protection = true
+  end
+
+  after(:all) do
+    Settings.sentry.dsn = nil
+    ActionController::Base.allow_forgery_protection = @original_val
+  end
+
+  before do
+    # innocuous route chosen for setting the CSRF token in the cookie
```

Comment:
```suggestion
    # innocuous route chosen for setting receiving the CSRF token in a header
```

Reply:
or something to keep the comment in line with the implementation

Reply:
fixed.  thanks.